src/transport-http-server.ts

```diff
@@ -7,17 +7,13 @@ import { Connection } from './connection.ts';
 
 import { logTransport2 as log } from './log.ts';
 
-import type { Opine } from '../deps.ts';
-
-const TIMEOUT = 1000; // TODO: make this configurable
```

Comment:
where is the general timeout defined now?

1 second is too short for production, but useful for testing

Reply:
I think the `TIMEOUT` exists only on the client, it's not needed in the server unless I'm mistaken.